metricq/subscription.py

```diff
@@ -51,11 +51,11 @@ def __init__(
             metrics (List[str], optional): List of metrics that you want to subscribe to.
             expires (Union[Timedelta, int, float]): The lifetime of the subscription queue in seconds.
         """
-        if not metrics:
-            raise ValueError("Metrics list must not be empty")
 
         super().__init__(*args, **kwargs)
-        self._metrics = metrics
+        self._metrics = list(metrics)
+        if not self._metrics:
+            raise ValueError("Metrics must not be empty")
```

Comment:
Again, that's technically change behaviour with the `TypeError` instead of `ValueError`